src/pm/hydra2/libhydra/bstrap/src/hydra_bstrap_common.h

```diff
@@ -13,7 +13,7 @@
 struct HYDI_bstrap_cmd {
     enum {
         HYDI_BSTRAP_CMD__PROXY_ID = 0,
-        HYDI_BSTRAP_CMD__HOSTLIST,
+        HYDI_BSTRAP_CMD__HOSTLIST
```

Comment:
It seems that C99 allows for the trailing comma in enum declarations. IMO, it makes a nicer git log to use the trailing comma and not have to modify multiple lines to add new values. Which compiler/flags are warning about this?

Reply:
I built using Intel Compiler 2019 Update 4 with -Wcheck option

Reply:
As far as I know C89 is default in Intel Compiler and gnu90 in gcc